contrib/passwd.bzl

```diff
@@ -60,11 +60,12 @@ passwd_file = rule(
             executable = True,
             allow_files = True,
         ),
+        "output_name": attr.string(default = "passwd"),
```

Comment:
Can we default to `%{name}`?

Reply:
this is changed now.
you can't default to name since attrs is not yet intiliazed and name is not available.